Jenkinsfile

```diff
@@ -1,426 +0,0 @@
-#!/usr/bin/env groovy
```

Comment:
Does this Jenkinsfile need to be removed? Was it removed for testing purposes and for this commit it should not be removed?

Reply:
Yes. 

This solution requires synchronization of the changes to both rocBLAS and Tensile. It is expected that the tests are going to fail and so this pr is to provide a review only request.

I approved the intent is to submit a valid pr which can get submitted quickly and merged, hopefully without breaking anything.